cors-proxy.js

```diff
@@ -0,0 +1,18 @@
+// server to proxy CORS requests for tunneled resources
+// via localhost port forwarding during development.
+
+// Listen on a specific host via the CPHOST environment variable
+var host = process.env.CPHOST || '127.0.0.1';
```

Comment:
Where would these environment variables get set? Do we have any documentation on when they should be set and what they should be set to?

Reply:
No, I basically added them for someone who didn't want to use the default localhost ip and port that I selected (which is what basically anyone would want).  The would be set when you run the command if you wanted something different.

```
CPHOST=#LocalIP PORT=#portnumber node cors-proxy.js
```